pikelet/src/pass/surface_to_core.rs

```diff
@@ -373,6 +380,10 @@ impl<'me> State<'me> {
     }
 
     /// Synthesize the type of a surface term, and return the elaborated term.
+    #[allow(clippy::suspicious_else_formatting)] // FIXME: Bug in `contracts`?
```

Comment:
Seems to be triggering [this clippy lint](https://rust-lang.github.io/rust-clippy/master/index.html#suspicious_else_formatting) too for some reason?

Reply:
This might be because of accidentally erased span information between the different attributes. See this issue: https://gitlab.com/karroffel/contracts/-/issues/8#note_339557232

If you add `use contracts::*;` and use unqualified versions of the attributes, does this still happen?

Reply:
Ahh! You are right! This seems to have fixed it!